styles.css

```diff
@@ -0,0 +1,339 @@
+  }
+  article{
+    grid-gap:5px !important;
+    width: 100% !important;
+  }
+  .row-1{
+    grid-gap:5px !important;
+  }
+  .col{
+    width: 100%;
+  }
+  .card-desc{
+    padding:2px !important;
+  }
+  .card-desc p{
+    padding: 0px !important;
+    font-size: small !important;
+  }
+  .footer{
+    width:100%;
+  }
+}
+/* Desktop Styles */
+@media only screen and (min-width: 960px) {
+  body{
+    width:100%;
+  }
+  header{
+    width:100%;
+  }
+  .main-container{
+    width:70%;
+  }
+  footer{
+    width:100%;
+  }
+  .banner-left-side{
+    width:50%;
+  }
+  .banner-right-side{
+    width:50%;
+  }
+}
+.main-container{
+  display: grid;
+  grid-template-rows: auto auto auto;
+  grid-gap: 30px;
+  margin : 20px 150px;
+}
+h4,h5{
+  margin:0px;
+}
+h3{
+  margin:3px;
+}
+p{
+  margin:0;
+  font-weight: 100;
+  font-size: small;
+}
+a{
+  color:white;
+  text-decoration: none;
+}
+header{
+  display: flex;
+  background-image: url('images/header-image.jpg');
+  height: 250px;
+  padding:40px;
+}
+.banner-left-side{
+  flex:80%;
+}
+.banner-left-side-img{
+  height:75px;
+}
+.banner-right-side{
+  flex:5;
+}
+.banner-right-side-cart{
+  display:flex;
+  justify-content: space-between;
+  align-items :baseline;
+  width: 80px;
+  height:25px;
+}
+.cart{
+  height:20px;
+}
+.cart-price{
+  font-size: smaller;
+  color:white;
+  margin: 0;
+}
+nav{
+  background-color: #e2e5e6;
+  height : 50px;
+}
+.nav-caption{
+  color:darkblue;
+  font-size:13px;
+  padding : 20px 150px;
+  font-family: serif;
+}
+nav a{
+  color:darkblue;
+}
+section{
+  display:flex;
+}
+.section-left-side{
+  flex:30%;
+}
+.section-left-side img{
+  height:230px;
+}
+.section-right-side{
+  flex: 70%;
+  padding:0px 150px 0px 30px;
+}
+.section-right-side-header{
+  color:darkblue;
+  font-family: serif;
+  font-size:x-large;
+}
+.section-right-side-para{
+  font-family: monospace;
+  padding: 0px 19px 0px 0px;
+  font-size:larger;
+}
+article{
+  display:grid;
+  grid-template-rows: auto auto;
+  grid-gap: 30px;
+}
+.row-1{
+display:grid;
+grid-template-columns: auto auto auto auto;
+grid-gap:100px;
+}
+.row{
+  display:grid;
+  grid-template-columns:auto auto auto;
+  grid-gap:20px;
+  font-size: large;
+}
+.col-1{
+  display:flex;
+  flex-direction:column;
+  flex-wrap:wrap;
+  font-size: large;
+}
+.col{
+  padding:0.5px;
+}
+.address-row{
+  display:grid;
+  grid-template-columns: 25% 25% 25%;
+  grid-gap: 15px;
+}
+.address-row-1{
+ display:flex;
+ flex-direction:column;
+ flex-wrap:wrap;
+ font-size: large;
+}
+.checkout-row{
+  display:grid;
+  grid-template-columns:auto auto;
+}
+.checkout-row-1{
+  display:grid;
+  grid-template-columns:15% 40%;
+}
+.checkout-row-1-1{
+  display:grid;
+  grid-template-rows:50% 50%;
+}
+.checkout-animals{
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  width: 50%;
+  height:60px;
+}
+
+.card{
+display:grid;
+grid-template-rows: auto auto auto;
+border: 0.5px solid;
+}
+.card-image{
+ height: 100px;
```

Comment:
Images are not being sized properly at all resolutions. A better adjustment might be 
`height: auto;`
`width: 100%;`